keyrings/alt/file.py

```diff
@@ -78,9 +81,9 @@ def priority(self):
             __import__('Crypto.Cipher.AES')
             __import__('Crypto.Protocol.KDF')
             __import__('Crypto.Random')
-        except ImportError:
+        except ImportError:     # pragma: no cover
```

Comment:
I'm not sure I understand the purpose of these pragmas. They feel like noise in the code. Plus, this code doesn't seem like code that we don't expect to be executed. It seems like code that's legitimately not covered by the tests.